File: python/examples/meeting_summary.py
# !pip install gladiaio-sdk
from gladiaio_sdk import GladiaClient

# Create your account and get your API key in 30 seconds at https://docs.gladia.io/chapters/introduction/getting-started
gladia_client = GladiaClient(api_key="GLADIA_API_KEY").prerecorded()

transcription = gladia_client.transcribe(
    audio_url="../data/meeting-example.mp4",
    # check all the summarization options at https://docs.gladia.io/chapters/audio-intelligence/summarization
    options={"summarization": True, "summarization_config": {"type": "bullet_points"}},
)

print(transcription.result.summarization)

File: typescript/examples/meeting-summary.ts
import { GladiaClient } from '@gladiaio/sdk';

const gladiaClient = new GladiaClient();
const result = await gladiaClient
  .preRecorded()
  .transcribe('../data/call-center-example.mp4', {
    summarization: true,
    summarization_config: {
      // check all the summarization options at https://docs.gladia.io/chapters/audio-intelligence/summarization
      type: 'bullet_points',
    },
  });

console.log(result.result?.summarization?.results ?? '');
